packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -61,13 +61,9 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
   }
 
   public componentWillReceiveProps(nextProps: IImageProps) {
+    // If the image is not loaded, recompute the cover style.
     if (this.state.loadState === ImageLoadState.loaded) {
```

Comment:
Does this if condition make sense? It seems like we should be checking if the image's source has changed to determine whether we need to computer the cover style.